daemon/logger/jsonfilelog/jsonfilelog.go

```diff
@@ -19,11 +22,12 @@ const (
 // JSONFileLogger is Logger implementation for default docker logging:
 // JSON objects to file
 type JSONFileLogger struct {
-	buf *bytes.Buffer
-	f   *os.File   // store for closing
-	mu  sync.Mutex // protects buffer
-
-	ctx logger.Context
+	buf      *bytes.Buffer
+	f        *os.File   // store for closing
+	mu       sync.Mutex // protects buffer
+	capacity int64
+	n        int
```

Comment:
What is `n` for?
I'm sure I can look and see, but would be helpful to know here in the struct definition.